_assets/compose/bootnode/Makefile

```diff
@@ -72,7 +72,7 @@ keys:
 
 start: keys/nodekey keys/nodeaddr
 	@echo " * $(GRN)Starting '$(CONTAINER_NAME)' container...$(RST)"
-	docker-compose up -d
+	docker-compose $(COMPOSE_START_FLAGS) up -d
```

Comment:
If anythin it should be `COMPOSE_UP_FLAGS` since they are for `docker-compose up` call.